scarb/src/core/config.rs

```diff
@@ -36,6 +38,7 @@ pub struct Config {
     offline: bool,
     compilers: CompilerRepository,
     cairo_plugins: CairoPluginRepository,
+    custom_source_patches: Option<Vec<ManifestDependency>>,
```

Comment:
Let's add a comment here and in actual usage place that this is Dojo-specific feature, to be removed when we'll decouple Dojo from Scarb-as-a-library 😃